src/js/background.js

```diff
@@ -532,16 +530,22 @@ Badger.prototype = {
    */
   updateBadge: function(tabId){
     if (!this.showCounter()){
-      chrome.browserAction.setBadgeText({tabId: tabId, text: ""});
+      if(chrome.browserAction.setBadgeText){
+        chrome.browserAction.setBadgeText({tabId: tabId, text: ""});
+      }
       return;
     }
     var numBlocked = this.blockedTrackerCount(tabId);
-    if(numBlocked === 0){
-      chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#00cc00"});
-    } else {
-      chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#cc0000"});
+    if(chrome.browserAction.setBadgeBackgroundColor){
+      if(numBlocked === 0){
+        chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#00cc00"});
+      } else {
+        chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#cc0000"});
+      }
+    }
+    if(chrome.browserAction.setBadgeText){
```

Comment:
Moved feature checks.

For what would you use that global variable? All my checks are just feature checks. The best way to determine if it's mobile is `chrome.runtime.getPlatformInfo((platformInfo) => {console.log(platformInfo.os)});`, but i don't see a specific use case.